pages/1.10/release-notes/1.10.6/index.md

```diff
@@ -6,15 +6,18 @@ menuWeight: 0
 excerpt:
 ---
 
-These are the release notes for DC/OS 1.10.6.
```

Comment:
Can we start adding release date?  Or at least release week?  So that we know that 1.10.6 was released approximately X date?

Reply:
@justinrlee, I've moved this question to my manager (Pavi) to respond. Thank you for your suggestion.